ansible/files/telemetry.sh

```diff
@@ -168,11 +172,41 @@ jupyter serverextension enable --py jupyter_spark --user
 jupyter nbextension install --py jupyter_spark --user
 jupyter nbextension enable --py jupyter_spark --user
 
+# Create analyses dir whether we are mounting efs or not
+mkdir -p $HOME/analyses
+
+if [ -n "$EMAIL" ]; then
+    # Mount entire efs locally to check for user's dir 
+    mkdir -p /mnt/efs
+    sudo mount -t nfs4 -o nfsvers=4.1,rsize=1048576,wsize=1048576,hard,timeo=600,retrans=2 fs-d0c30f79.efs.us-west-2.amazonaws.com:/ /mnt/efs
+
+    if [ ! -d "/mnt/efs/$EMAIL" ]; then
+        # If no dir for this user, create it, set ownership, add examples
+        sudo mkdir -p "/mnt/efs/$EMAIL"
+
+        sudo chown hadoop:hadoop "/mnt/efs/$EMAIL"
+
+        wget -nc -P "/mnt/efs/$EMAIL/" https://raw.githubusercontent.com/mozilla/emr-bootstrap-spark/master/examples/Telemetry%20Hello%20World.ipynb
```

Comment:
@vitillo I think that's on the wishlist right now, see [0] and [1]. [2] requests a new file extension for notebook templates, which would only ever open as copies of the original - exactly what we want (_/me_ _swoons_).

[0] StackOverflow question about default notebook: http://stackoverflow.com/questions/34002326/default-template-for-ipython-notebook-using-jupyter/34075215
[1] Backlog Item for Jupyter: https://github.com/jupyterlab/jupyterlab/issues/1390
[2] Template Notebook issue: https://github.com/jupyter/notebook/issues/332